ios/PurchasesHybridCommon/PurchasesHybridCommon/CommonFunctionality.swift

```diff
@@ -754,6 +754,56 @@ import RevenueCat
     }
 }
 
+// MARK: - Redemption links
+@objc public extension CommonFunctionality {
+
+    @objc(isWebPurchaseRedemptionURL:)
+    static func isWebPurchaseRedemptionURL(urlString: String) -> Bool {
+        guard let url = URL(string: urlString) else { return false }
+
+        return url.asWebPurchaseRedemption != nil
+    }
+
+    static func redeemWebPurchase(urlString: String,
+                                  completion: @escaping ([String: Any]?, ErrorContainer?) -> Void) {
```

Comment:
Just a question, do we not need this from ObjC? Or does it take the annotation from the `extension`?

Reply:
Good question... it worked when I tested it in RN, so it seems it's not needed... But going to add the annotation to be consistent. Thanks!